payments/management/commands/delete_discount_code.py

```diff
@@ -0,0 +1,34 @@
+from django.core.management import BaseCommand
+
+from payments.models import Discount
+
+
+class Command(BaseCommand):
+    """
+    Deletes multiple discounts using the Discount IDs.
+    An example usage of this command: python manage.py delete_discount_code 1 2 3
+    """
+
+    help = "Deletes multiple discounts using the Discount IDs."
+
+    def add_arguments(self, parser) -> None:
+        """
+        Add arguments to the command parser.
+        """
+        parser.add_argument(
```

Comment:
This needs to be the code itself, and should include the ability to deactivate a batch.